devops/scripts/rebase-ci.sh

```diff
@@ -14,8 +14,8 @@ if [[ -n "${CIRCLE_PULL_REQUEST:-}" ]]
 then
     GITHUB_PR_URL="https://api.github.com/repos/freedomofpress/securedrop/pulls/${CIRCLE_PULL_REQUEST##*/}"
 
-    # If current branch is not master or doesnt start with release...
-    if [[ "$CIRCLE_BRANCH" != "master"  && "$CIRCLE_BRANCH" != release*  ]]; then
+    # If current branch is not main or doesnt start with release...
+    if [[ "$CIRCLE_BRANCH" != "main"  && "$CIRCLE_BRANCH" != release*  ]]; then
```

Comment:
1. If we keep this `if` clause, should it say `stable`, or drop the `main` condition entirely, since no `main` branch exists in this repo?
2. Does this clause still make sense? It looks like we've carried it over from when Mike first implemented this back in #2298, which hardcoded a rebase against `develop`. That was subsequently fixed in #3617, and we're now always rebasing against the target branch. What does this check, now, accomplish?

Reply:
That's a fair point, and there's no need to skip it for release branches either AFAICT. removed.